src/select.jsx

```diff
@@ -344,7 +344,7 @@ var classBase = React.createClass({
           </div>
           : ''
         }
-        <select name={this.props.selectName} value={this.state.selectedOptionVal} className={this.props.hiddenSelectClassName} tabIndex={-1} aria-hidden={true} >
+        <select name={this.props.selectName} onChange={() => {}} value={this.state.selectedOptionVal} className={this.props.hiddenSelectClassName} tabIndex={-1} aria-hidden={true} >
```

Comment:
`readOnly` sounds like the right thing here. The user never changes the `select` element directly.